tests/omise/EventTest.php

```diff
@@ -0,0 +1,36 @@
+<?php require_once dirname(__FILE__).'/TestConfig.php';
+
+class EventTest extends TestConfig {
+  /**
+   * OmiseEvent class must be contain methods as below.
+   *
+   */
+  public function testMethodExists() {
+    $this->assertTrue(method_exists('OmiseEvent', 'retrieve'));
+    $this->assertTrue(method_exists('OmiseEvent', 'reload'));
+    $this->assertTrue(method_exists('OmiseEvent', 'getUrl'));
+  }
+
+  /**
+   * Assert that a list of event object could be successfully retrieved.
+   *
+   */
+  public function testRetrieveEventListObject() {
+    $event = OmiseEvent::retrieve();
```

Comment:
@nimid @guzzilar I got it, my bad. I ran phpunit from Ubuntu it doesn't work but when I run phpunit from composer, it works fine, just realized it's different.